xml/net_samba.xml

```diff
@@ -778,13 +730,74 @@
     configuration.
    </para>
   </sect2>
+  <sect2 xml:id="sec.samba.client.old-server">
+   <title>Enabling Clients to Work With SMB1 Servers</title>
+   <para>
+    The first version of the SMB network protocol, SMB1, is an old and
+    insecure protocol which has been deprecated by its originator, Microsoft.
+    For security reasons, &productname; now expects shares to use the
+    protocol version SMB 2.1, 3.0, or 3.02 by default.
```

Comment:
maybe rephrase to: 'one of the following protocol versions by default: SMB 2.1, 3.0, or 3.02.'

Reply:
I rephrased a bit differently...